go.mod

```diff
@@ -6,10 +6,12 @@ require (
 	github.com/davecgh/go-spew v1.1.1
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-bindata/go-bindata v3.1.2+incompatible
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
```

Comment:
since we already have klog can we use that instead?

Reply:
Yes. Thanks. I got rid of glog.